back-end/controllers/fixtures_byId.controller.js

```diff
@@ -0,0 +1,15 @@
+const { fetchFixturesById } = require("../models/fixtures_byId.model.js");
+
+const getFixturesById = (req, res, next) => {
+  const { fixture_id } = req.params;
+  fetchFixturesById(fixture_id)
+    .then((fixture) => res.status(200).send({ fixture }))
+    .catch((err) => {
```

Comment:
We capture the error in app.js, so you only need .catch(next) instead

Reply:
removed